src/tr1/math/matrix.h

```diff
@@ -20,7 +20,7 @@ void Matrix_RotX(int16_t rx);
 void Matrix_RotY(int16_t ry);
 void Matrix_RotZ(int16_t rz);
 void Matrix_RotYXZ(int16_t ry, int16_t rx, int16_t rz);
-void Matrix_RotYXZpack(int32_t rots);
+void Matrix_RotYXZ_16(const XYZ_16 *rotation);
```

Comment:
Let's just call it Rot or RotXYZ as the order doesn't matter anymore.

Reply:
I'm also ok with XYZ16, as underscores are used in a different way in our function names, and this nane would clearly distinguish this function's signature from, say, TranslateRel which take individual components still.

Reply:
I've gone with `Matrix_RotXYZ16`, agreed it keeps it clear.